db/db_range_del_test.cc

```diff
@@ -3465,13 +3465,12 @@ TEST_F(DBRangeDelTest, NonBottommostCompactionDropRangetombstone) {
                              Key(5)));
   CompactRangeOptions cro;
   cro.bottommost_level_compaction = BottommostLevelCompaction::kForceOptimized;
-  end_str = Key(5);
-  end_slice = end_str;
-  ASSERT_OK(db_->CompactRange(cro, &begin_slice, &end_slice));
-  ASSERT_EQ(NumTableFilesAtLevel(1), 1);
+  ASSERT_OK(db_->CompactRange(cro, nullptr, nullptr));
+  // All compacted to L6
+  ASSERT_EQ("0,0,0,0,0,0,1", FilesPerLevel(0));
   db_->GetProperty(DB::Properties::kAggregatedTableProperties, &property);
   ParseTablePropertiesString(property, &output_tp);
-  ASSERT_EQ(output_tp.num_range_deletions, 2);
+  ASSERT_EQ(output_tp.num_range_deletions, 1);
```

Comment:
The DeleteRange after the snapshot is logically redundant, so this is testing a behavior that is peculiar to the implementation rather than necessary for data correctness. Not ideal. That could be fixed by Put on key 4 before the snapshot.

Reply:
Fixed by adding Put on key 4.